Templates/SaaSFileTrigger-Batch/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "type": "apiHubFileTrigger",
+      "name": "input",
+      "direction": "in",
+      "path": "input-path-batch/{name}",
```

Comment:
It just adds to confusion. people will try creating different functions for different languages in the same function app and would not see the triggers being fired because it might have been picked up by another trigger function.
Does that make sense?